src/makeChange.js

```diff
@@ -1,5 +1,61 @@
 'use strict'
 
 export default function makeChange({price, amountGiven}) {
-  // your code here
-}
+};
+
+// make change Logic
+let quarters = 0;
+  let dimes = 0;
+  let nickels = 0;
+  let pennies = 0;
+
+var calculateChange = function(amountPaid) {
+let change = amountPaid * 100;
+
+  switch (true) {
+  case (change >= 25):
+    quarters += 1;
+    let amountPaid1 = (change - 25) / 100;
+  calculateChange(amountPaid1);
+  break;
+  case (change >= 10):
+    dimes += 1;
+    let amountPaid2 = (change - 10)/ 100;
+  calculateChange(amountPaid2);
+  break;
+  case (change >= 5):
+    nickels  += 1;
+    let amountPaid3 = (change - 5)/ 100;
+  calculateChange(amountPaid3);
+  break;
+  case (change >= 1):
+    pennies  += 1;
```

Comment:
if theres still a problem with calculating pennies, i wonder if its occuring in the recursion